File: ios/GroceryWatch/GroceryWatch Watch App/GroceryItem.swift
import Foundation

struct GroceryItem: Identifiable, Codable {
    var id: String?
    var name: String
    var isCompleted: Bool
    var order: Int
    var createdAt: Date
    
    init(id: String? = nil, name: String, isCompleted: Bool = false, order: Int = 0, createdAt: Date = Date()) {
        self.id = id
        self.name = name
        self.isCompleted = isCompleted
        self.order = order
        self.createdAt = createdAt
    }
}

File: android/app/src/main/java/com/example/grocery/model/GroceryItem.kt
package com.example.grocery.model

import com.google.firebase.firestore.DocumentId
import com.google.firebase.firestore.PropertyName
import java.util.Date

data class GroceryItem(
    @get:DocumentId
    var id: String = "",
    
    var name: String = "",
    
    @get:PropertyName("isCompleted")
    @set:PropertyName("isCompleted")
    var isCompleted: Boolean = false,
    
    var order: Int = 0,
    
    var createdAt: Date = Date()
)
